test/model/recent_senders_test.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/model/recent_senders.dart';
+import '../example_data.dart' as eg;
+
+void main() {
+  group('MessageIdTracker', () {
+    late MessageIdTracker idTracker;
+
+    void prepare() {
+      idTracker = MessageIdTracker();
+    }
```

Comment:
This can just be inlined in each test case:
```dart
      final idTracker = MessageIdTracker();
```

Still just one line, and makes each test more explicit and self-contained.